rotkehlchen/interfaces.py

```diff
@@ -25,9 +25,16 @@ def query_current_price(
             self,
             from_asset: AssetWithOracles,
             to_asset: AssetWithOracles,
-    ) -> Price:
-        """Returns the price from_asset to to_asset at the current timestamp
+            match_main_currency: bool,
+    ) -> Tuple[Price, bool]:
+        """
+        Accepts a pair of assets to find price for and a flag. If `match_main_currency` is True
+        and there is a manual latest price that has value in `main_currency`, then it will be
+        returned without the conversion to `to_asset`.
+        Returns:
+        1. The price of from_asset at the current timestamp
         for the current oracle
+        2. Whether returned price is in `to_asset` or in main currency
```

Comment:
```suggestion
        2. Whether returned price is in main currency
```

This is more certain. Otherwise the docstring (just like the code) is not clear on what happens if to_asset == main_currency